src/deleteEmployee.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  originalData,
+  fetchData,
+  fillentry,
+  firebaseData,
+  toast,
+} from "./script";
+import { api, skillList, overlay, dataDelModal } from "./constants";
+const confirmButton = document.querySelector(
+  ".confirm-button"
+)! as HTMLButtonElement;
+const delData = (index: number) => {
+  fetch(api + "/employee/" + index + ".json", {
+    method: "DELETE",
+  })
+    .then((res) => {
+      console.log(res, "successfully deleted!!!");
```

Comment:
Remove all unnecessary console logs. Except for getting error message codes in catch block you can remove other console logs